distribution/ddf-common/src/main/resources/etc/ws-security/issuer/encryption.properties

```diff
@@ -3,5 +3,5 @@ org.apache.ws.security.crypto.provider=org.apache.wss4j.common.crypto.Merlin
 org.apache.ws.security.crypto.merlin.keystore.type=jks
 org.apache.ws.security.crypto.merlin.keystore.password=${javax.net.ssl.keyStorePassword}
 org.apache.ws.security.crypto.merlin.keystore.private.password=${javax.net.ssl.keyStorePassword}
-org.apache.ws.security.crypto.merlin.keystore.alias=${org.codice.ddf.system.hostname}
+org.apache.ws.security.crypto.merlin.keystore.alias=${org.codice.ddf.external.hostname}
```

Comment:
❗️ These four should not be external.

Reply:
I definitely had to change them to external to get things to work, granted this might be more due to how the `CertNew` tool loads the cert into the keystore than anything since it assumes the value provided to the `-cn <fqdn>` option should be used as the keystore alias

Reply:
I think it was because some other properties got refactored to `EXTERNAL` when they should not have been. 😸